src/corehost/common/longfile.windows.cpp

```diff
@@ -65,6 +69,19 @@ bool AssertRepeatingDirSeparator(const pal::string_t& path)
 
     assert(path_to_check.find(altDirSeparator) == pal::string_t::npos);
 
+    pal::string_t combDirSeparator1;
+    combDirSeparator1.push_back(LongFile::DirectorySeparatorChar);
+    combDirSeparator1.push_back(LongFile::AltDirectorySeparatorChar);
+
+    assert(path_to_check.find(combDirSeparator1) == pal::string_t::npos);
```

Comment:
The above asserts are to ensure bad path manipulations are not done. These will largely work for short paths, but if the extended prefix is added then they will not work